Dockerfile

```diff
@@ -11,13 +11,18 @@ COPY . .
 RUN go mod download
 RUN go mod verify
 RUN CGO_ENABLED=0 GOOS=linux go build -o /go/bin/app -ldflags="-s -w" -v ./cmd/
+# Create licenses directory (for local builds when it doesn't exist)
+# licenses.json for 3rd party licenses is generated during the release workflow
+RUN mkdir -p /go/src/app/licenses
```

Comment:
This doesnt make sense, if the workflow created the directory beforehand, why do we need this mkdir?